niworkflows/viz/utils.py

```diff
@@ -56,7 +59,7 @@ def as_svg(image):
 
 def cuts_from_bbox(mask_nii, cuts=3):
     from nibabel.affines import apply_affine
-    imnii = nb.load(mask_nii)
+    imnii = mask_nii
```

Comment:
A loaded image is now being passed to cuts_from_bbox Line 128:
`mask_nii = nb.load(mask_nii) if masked else nlimage.threshold_img(mask_nii, 1e-3)`